### PR TITLE
THREESCALE-3046: Fix policy jsonschema.

### PR DESCRIPTION
**Pull request by eloycoto**
Fix various typos.
Added some required fields.
Add description to operations.

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

### CODE REVIEW

**Comment by davidor** [comment]
Should we change the base branch to `3.6-stable` ? That way we could include it in the candidate release next week.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - Introduce possibility of specifying policy order restrictions in their schemas. APIcast now shows a warning when those restrictions are not respected [#1088](https://github.com/3scale/APIcast/pull/1088), [THREESCALE-2896](https://issues.jboss.org/browse/THREESCALE-2896)
+- Fix typos on JWT claim policy jsonschema [THREESCALE-3046](https://issues.jboss.org/browse/THREESCALE-3046)
```
Comment (davidor) [inline CHANGELOG.md]: I think we should move this to a "Fixed" sub-section.

File: gateway/src/apicast/policy/jwt_claim_check/apicast-policy.json [hunk 2]
```diff
@@ -22,6 +22,9 @@
         }]
       }
     },
+    "required": [
+      "error_message"
```
Comment (davidor) [inline gateway/src/apicast/policy/jwt_claim_check/apicast-policy.json]: Why is the error message required? Shouldn't we have a default one?

File: CHANGELOG.md (hunk 1 above)
Comment (eloycoto) [reply CHANGELOG.md]: :man_facepalming: true

File: gateway/src/apicast/policy/jwt_claim_check/apicast-policy.json (hunk 2 above)
Comment (eloycoto) [reply gateway/src/apicast/policy/jwt_claim_check/apicast-policy.json]: true

**Pull request merged**